### ISSUE TITLE
Connect Inoreader without API Key creation requirement

### ISSUE THREAD

**Issue by hcaginlenk** [yang991178/fluent-reader#575]
**Is your feature request related to a problem? Please describe.**
I do not have a pro subscription to Inoreader and, as far as I am aware, I can not create my own API key with a free plan.

**Describe the solution you'd like**
API key input should be optional, not a requirement to be able to connect Inoreader with Fluent Reader. Connecting Inoreader to mobile apps with a free plan is not causing any problem for my use case.

**Comment by a15355447898a** [comment]
The same ideas should not be limited to connecting only through APIs with Inoreader. There should be choices available instead of being constrained.

**Comment by JupiterOdyssey** [comment]
Agree with that. The feature is essential.

**Comment by Chikit-L** [comment]
I agree with your point. When using Inoreader on the Reeder app, I don't need an API, and I haven't encountered any issues while using it.

**Comment by bootlegcoal** [comment]
Agree.
